step-templates/octopus-apply-octoterra-module-s3.json

```diff
@@ -45,8 +45,8 @@
       "Id": "27254625-8cfd-4918-b16b-68ac26a25d37",
       "Name": "OctoterraApply.Terraform.Workspace.Name",
       "Label": "Terraform Workspace",
-      "HelpText": "The name of the Terraform workspace. This must be unique for every project this module is deployed to. The default value is based on the space name: `#{Octopus.Space.Name | Replace \"[^A-Za-z0-9]\" \"_\"}`",
-      "DefaultValue": "#{Octopus.Space.Name | Replace \"[^A-Za-z0-9]\" \"_\"}",
+      "HelpText": "The name of the Terraform workspace. This must be unique for every project this module is deployed to. The default value is based on the space ID that the module is applied to: `#{OctoterraApply.Octopus.SpaceID}`. Leave this as the default value unless you have a specific reason to change it.",
```

Comment:
The value of this parameter is defaulted to another parameter that doesn't have a default value, I'm assuming this is intentional?

Reply:
Yes, 99% of the time this value is just the space ID. There will be edge cases where this value is not unique enough, usually when applying values across multiple instances that may have duplicate space IDs, which is why it is a configurable field. But the default is to reuse an internal parameter.